app/views/vendors/_review_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<form>
+  <div class="row collapse">
+    <h3>Add Review:</h3>
+    <%= form_for [@vendor, @review] do |f| %>
+    <%= f.label :radio_button, "Rating" %>
+    <%= f.radio_button(:rating, "1") %>
+    <%= f.label(:rating_1, "1") %>
+    <%= f.radio_button(:rating, "2") %>
+    <%= f.label(:rating_2, "2") %>
+    <%= f.radio_button(:rating, "3") %>
+    <%= f.label(:rating_3, "3") %>
+    <%= f.radio_button(:rating, "4") %>
+    <%= f.label(:rating_4, "4") %>
+    <%= f.radio_button(:rating, "5") %>
+    <%= f.label(:rating_5, "5") %>
+
+    <%= f.label :body, "Comments:" %>
+    <%= f.text_field :body %>
+
+    <%= f.submit %>
+    <% end %>
```

Comment:
This might be useful: http://apidock.com/rails/v4.0.2/ActionView/Helpers/FormOptionsHelper/collection_radio_buttons

Also, everything within the `form_for` block should be indented.